kea-vscode/src/core/commit-file-content-provider.test.ts

```diff
@@ -0,0 +1,155 @@
+import * as assert from "assert";
+import * as sinon from "sinon";
+import * as vscode from "vscode";
+import { CommitFileContentProvider } from "./commit-file-content-provider";
+import { createKeaContextStub } from "../test-utils";
+import { LocalGitRepository } from "../git/local-git-repository";
+
+suite("CommitFileContentProvider", () => {
```

Comment:
Please fix all the broken tests in this folder. Make sure to compile the tests before running them.

Reply:
These tests are still broken:

```
TypeError: Cannot stub non-existent property getFileAtCommit
```